db-tools-i18n/src/main/resources/com/splicemachine/db/loc/messages.xml

```diff
@@ -9000,6 +9000,21 @@ Shutting down instance {0} on database directory {1} with class loader {2} </tex
                <arg>externalAttributeCount</arg>
            </msg>
 
+           <msg>
+               <name>EXT25</name>
+               <text>INSERT in a PIN TABLE is not allowed</text>
+           </msg>
+
+           <msg>
+               <name>EXT26</name>
+               <text>UPDATE a PIN TABLE is not allowed</text>
+           </msg>
+
+           <msg>
+               <name>EXT27</name>
+               <text>DELETE in a PIN TABLE is not allowed</text>
```

Comment:
Juste verified and TRUNCATE doesn't not support splice-properties . The user get an error message whenever he will try to do that so we are good.